Sources_v3/StreamChatUI/ChatChannel/MessageComposerInputAccessoryViewController.swift

```diff
@@ -38,32 +74,99 @@ open class MessageComposerInputAccessoryViewController<ExtraData: UIExtraDataTyp
     override open func viewDidLoad() {
         super.viewDidLoad()
         
+        setUp()
         setupInputView()
         setUpLayout()
+        updateContent()
+    }
```

Comment:
Please use _exactly_ the snippet from `ViewController`. We have to make sure the behavior is exactly the same everywhere.